packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -58,7 +59,7 @@ Style guide: components.table.react-tablehead
 /*
 `<TableBody>`
 
-`TableBody` renders the `<tbody>` element and contains table rows defining the table data.
+`TableBody` renders the `<tbody>` element and will typically contain `TableRow` elements to define table data.
 
 @react-props TableBody.jsx
```

Comment:
Remove `@react-props TableBody.jsx` and other `@react-prop` sections for `<TableCaption>, <TableHead>, <TableBody>, <TableRow>`

```suggestion

```

Reply:
If I remove that, then the link for `View source file` will disappear (see attached image further up).